frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSettings.tsx

```diff
@@ -1,36 +1,38 @@
 import type { ChangeEvent } from "react";
 import { t } from "ttag";
 
-import { useSelector } from "metabase/lib/redux";
-import { Box, Stack } from "metabase/ui";
+import CS from "metabase/css/core/index.css";
+import { Box, Stack, Text } from "metabase/ui";
 
-import { getSettings } from "../../selectors";
 import SettingHeader from "../SettingHeader";
-import { SetByEnvVarWrapper } from "../SettingsSetting";
 import {
   EmbeddingSdkOptionCard,
   InteractiveEmbeddingOptionCard,
   StaticEmbeddingOptionCard,
 } from "../widgets/EmbeddingOption";
-import { EmbeddingSwitchWidget } from "../widgets/EmbeddingSwitchWidget";
 
 import type { AdminSettingComponentProps } from "./types";
 
 export function EmbeddingSettings({
   updateSetting,
 }: AdminSettingComponentProps) {
-  const enableEmbeddingSetting = useSelector(getSettings).find(
-    (setting: any) => setting.key === "enable-embedding",
-  );
-
-  function handleChangeEnableEmbedding(value: boolean) {
-    updateSetting({ key: "enable-embedding" }, value);
+  function handleToggleStaticEmbedding(event: ChangeEvent<HTMLInputElement>) {
```

Comment:
these new handlers are for each toggle in each embedding card
![image](https://github.com/user-attachments/assets/8410699a-659f-4822-a212-344af65fb531)